src/plugins/kibana_legacy/jest.config.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
   preset: '@kbn/test',
   rootDir: '../../..',
   roots: ['<rootDir>/src/plugins/kibana_legacy'],
+  coverageDirectory: '<rootDir>/target/kibana-coverage/jest/src/plugins/kibana_legacy',
+  coverageReporters: ['text', 'html'],
+  collectCoverageFrom: ['<rootDir>/src/plugins/kibana_legacy/{public,server}/**/*.{js,ts,tsx}'],
```

Comment:
nit: kibana_legacy plugin doesn't have a server folder

Reply:
thank you, I will push an update

Reply:
fixed in 272fc2911860f0e0d9e066152fc8fb5dda97b3a0